contributing_to_docs/doc_guidelines.adoc

```diff
@@ -1,39 +1,30 @@
-[[contributing-to-docs-doc-guidelines]]
+[id='contributing-to-docs-doc-guidelines']
 = Documentation guidelines
-:icons:
-:toc: macro
-:toc-title:
-:toclevels: 1
-:description: These are basic guidelines for creating technical documentation for OpenShift 4.x.
+include::modules/common-attributes.adoc
 
 NOTE: The guidelines in this branch are specific for OpenShift / OKD 4.x
 documentation. See the
 link:https://github.com/openshift/openshift-docs/tree/master/contributing_to_docs[`master` branch]
-for guidelines for OpenShift / OKD 3.x documentation.
+for guidelines for OpenShift / OKD 3.x documentation. These guidelines build on top of the
```

Comment:
"These guidelines build on top of the link:https://redhat-documentation.github.io/modular-docs/[Red Hat modular docs reference guide]." needs to be in a separate paragraph.